liblangutil/Scanner.cpp

```diff
@@ -331,6 +331,11 @@ int Scanner::scanSingleLineDocComment()
 			{
 				addCommentLiteralChar('\n');
 				m_char = m_source->advanceAndGet(3);
+				if (tryScanEndOfLine())
```

Comment:
Doesn't this ignore `m_char`?

Reply:
@chriseth No. It just checks if `m_char` is `\n` or `\r` or `\r\n`.

Reply:
Ok, but why `break`?

Reply:
I mean what happens if there is another `///` on the next line?

Reply:
Without the break, in line 348, the scanner would advance to the next line which will cause problem. 

There won't be any compilation problems if another `///`. It will be treated as a new docstring block. Let me think if we should include it as part of the previous one.

Reply:
We should not split up docstring blocks, unless they are separated by an empty line (is that the case?)